### PR TITLE
Removed waits from realm test

### PR DESCRIPTION
**Pull request by edewit**
Now uses the new faster login method + removed all wait and uses the test realm it creates

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@edewit looks like there is a test failure, is it related to your changes?

File: cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts [hunk 1]
```diff
@@ -772,9 +777,9 @@ export default class RealmSettingsPage {
   shouldCancelEditingExecutor() {
     cy.get(this.clientProfileTwo).click();
 
-    cy.intercept("/auth/admin/realms/master/client-policies/profiles*").as(
-      "profilesFetch"
-    );
+    cy.intercept(
+      `/auth/admin/realms/${this.realmName}/client-policies/profiles*`
```
Comment (jenny-s51) [inline cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts:781]:
```suggestion
      `/auth/admin/realms/${this.realmName}/clientPolicies/profiles*`
```

File: cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts [hunk 2]
```diff
@@ -1117,9 +1128,16 @@ export default class RealmSettingsPage {
     cy.findByTestId(this.newClientPolicyDescriptionInput).type(
       "Test Again Description"
     );
+    cy.intercept(
+      "PUT",
+      `/auth/admin/realms/${this.realmName}/client-policies/policies`
```
Comment (jenny-s51) [inline cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts:1133]:
```suggestion
      `/auth/admin/realms/${this.realmName}/clientPolicies/policies`
```

**Comment by jonkoops** [comment]
@edewit can you rebase this one on `main`?

File: cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts (hunk 1 above)
Comment (edewit) [reply cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts:781]: no @jenny-s51 it's `client-policies` not `clientPolicies` see: https://github.com/keycloak/keycloak-nodejs-admin-client/blob/main/src/resources/clientPolicies.ts#L13

**Review by jonkoops — CHANGES_REQUESTED** [review]
Still seeing some test failures

**Review by jonkoops — APPROVED** [review]
LGTM

**Pull request merged**